### PR TITLE
Implement data render

### PR DESCRIPTION
**Pull request by thakichow**
@ganjihyok take a look when you have the chance!

### CODE REVIEW

File: client/components/Description.jsx [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+      {descriptionBody.split('\n').map(
+        par => (
+          <p>
+            {par}
+          </p>
+        ),
+      )}
+    </div>
+
+    <span role="button" tabIndex="0" className="read-more" onClick={() => { toggle(); }} onKeyPress={() => { toggle(); }}>
```
Comment (ganjihyok) [inline client/components/Description.jsx]: what is the purpose of onKeyPress on <span> tag?

File: client/components/HouseRules.jsx [hunk 2]
```diff
@@ -1,9 +1,60 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const HouseRules = () => (
-  <div className="HouseRules">
-    HouseRules
+const HouseRules = ({ rules, showHouseRules, toggleHouseRules }) => (
+  <div className="house-rules">
```
Comment (ganjihyok) [inline client/components/HouseRules.jsx]: unnecessary console.log here

File: client/components/HouseRules.jsx [hunk 3]
```diff
@@ -1,9 +1,60 @@
...
+              Read all rules
+            </span>)
+          : (
+            <span className="bold">
+              Hide ^
+            </span>
+          )
+      }
+    </span>
+    <div className="divider" />
```
Comment (ganjihyok) [inline client/components/HouseRules.jsx]: You can try using css border to create a bottom divider line! 
Just saying there's different way to do it.

File: client/components/Description.jsx (hunk 1 above)
Comment (thakichow) [reply client/components/Description.jsx]: For a non-button clickable element, eslint solicites a keyboard interaction option for accessibility purposes.

**Pull request merged**